readme.md

```diff
@@ -1001,6 +1001,14 @@ If you're looking to replicate the experience of taking acid without actually do
 
 > A bureaucrat in a retro-future world tries to correct an administrative error and himself becomes an enemy of the state.
 
+#### [Cargo](http://www.imdb.com/title/tt0381940/) (2009) [6.2]
+
+Categories: `space` `dystopia`
+
+Why not try something out of the ordinary with this Swizz sci-fi thriller? [@Lekoaf](https://github.com/lekoaf)
```

Comment:
Should this be Swiss? 

Would be great to add a note on why _you_ think this is awesome. Also, need a dash before your name.